tests/upgrades/test_classparameter.py

```diff
@@ -52,28 +66,39 @@ class TestScenarioPositivePuppetParameterAndDatatypeIntact:
     """
 
     @pytest.fixture(scope="class")
-    def _setup_scenario(self, target_sat, save_test_data):
+    def _setup_scenario(self, class_target_sat):
         """Import some parametrized puppet classes. This is required to make
         sure that we have smart class variable available.
         Read all available smart class parameters for imported puppet class to
         be able to work with unique entity for each specific test.
         """
-        self.org = entities.Organization().create()
+        enable_satellite_cmd = InstallerCommand(
+            installer_args=[
+                'enable-foreman-plugin-puppet',
+                'enable-foreman-cli-puppet',
+                'enable-puppet',
+            ],
+            installer_opts=common_opts,
+        )
+        result = class_target_sat.execute(enable_satellite_cmd.get_command(), timeout='20m')
+        assert result.status == 0
+        class_target_sat.execute('hammer -r')
```

Comment:
This seems like a workaround for BZ#2039696. If so, can you mention it in a comment?
Also see #10721